pkg/cli/admin/compare/compare.go

```diff
@@ -0,0 +1,394 @@
+    	to the user.
+		
+		The input cluster configuration may be provided as an "offline" set of CRs or can be pulled from a live cluster.
+		
+		The Reference also includes a mandatory metadata.yaml file where all the Resource templates should be specified.
+		The Resource templates can be divided into components. Each component and Resource template can be set as required,
+		resulting in a report to the user in case one of them is missing.
+		
+		Each Resource definition should be in its own template file. 
+		The usage of all Go built-in functions is supported along with the functions in the Sprig library.
+		All templates should always be valid YAML after template execution, even when injecting an empty mapping.
+		Before using functions that can fail for nil values, always check that the value exists.
+		
+		Each Resource definition should be in a separate template file, and the use of all Go built-in functions, 
+        along with those in the Sprig library, is supported. It is important that all templates remain valid YAML 
+		after execution, even in scenarios where an empty mapping is injected. 
+		Prior to using functions that may encounter nil values, it is critical to verify the existence of the value.
+
+		It's possible to pass a user config that contains an option to specify manual matches between cluster resources
+		and Resource templates. The matches can be added to the config as pairs of 
+		apiVersion-kind-namespace-name: <Template File Name>.
+
+		KUBECTL_EXTERNAL_DIFF environment variable can be used to select your own diff
+		command. Users can use external commands with params too, example:
+		KUBECTL_EXTERNAL_DIFF="colordiff -N -u"
+		
+		 By default, the "diff" command available in your path will be run with the "-u"
+		(unified diff) and "-N" (treat absent files as empty) options.
+		
+		 Exit status: 0 No differences were found. 1 Differences were found. >1 oc
+		or diff failed with an error.
+		
+		 Note: KUBECTL_EXTERNAL_DIFF, if used, is expected to follow that convention.
+
+		Experimental: This command is under active development and may change without notice.
+	`)
+
+	compareExample = templates.Examples(`
+		# Compare a known valid reference configuration with a live cluster:
+		oc adm compare -r ./reference
+		
+		# Compare a known valid reference configuration with a local set of CRs:
+		oc adm compare -r ./reference -f ./crsdir -R
+
+		# Compare a known valid reference configuration with a live cluster and with a user config:
+		oc adm compare -r ./reference -c ./user_config
+	`)
+	fieldGroups = [][][]string{
+		{{"apiVersion"}, {"metadata", "name"}, {"metadata", "namespace"}, {"kind"}},
+		{{"apiVersion"}, {"metadata", "name"}, {"kind"}},
+		{{"metadata", "namespace"}, {"kind"}},
+		{{"kind"}},
+	}
+)
+
+const (
+	ReferenceFileName        = "metadata.yaml"
+	noReffDirectoryWasPassed = "\"Reference directory is required\""
+	reffDirNotExistsError    = "\"Reference directory doesnt exist\""
```

Comment:
Looking at other "oc" error messages I think you'll have to lowercase + remove the `\"`